gandalf.js

```diff
@@ -1,19 +1,20 @@
 /**
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.